### PR TITLE
Saturday Summary 4

### PR DESCRIPTION
**Pull request by Muirrum**

### CODE REVIEW

**Comment by Muirrum** [comment]
Waiting on at least @cedric-h to respond to the questions in dev chat

File: content/blog/saturday-summary-4.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+the dev team we want to see that community grow. Nick would love to see more
+people on our chat software (which you can find at
+[https://chat.hackagotch.io](https://chat.hackagotch.io)). Owen wants to see the
+community start to create their own tools and spaces both on and off of the game
+chat, and is willing to give those projects the support they need.
+
+## Conclusion
+Well, that's it for this week's Saturday Summary! I'd like to get these out more
+regularly. If you have any questions feel free to drop a note in #hackstead on
+the Hack Club Slack or join us on IRC!
```
Comment (nzeng8) [inline content/blog/saturday-summary-4.md]: _Perhaps include a link to chat.hackagotch.io?_

Comment (Muirrum) [reply content/blog/saturday-summary-4.md]: line 45 in the Question 3 section

**Review by lukec11 — APPROVED** [review]
lgtm

**Pull request merged**